### ISSUE TITLE
Error in repy_sync

### ISSUE THREAD

**Issue by aschwanb** [hjuutilainen/sus-inspector#10]
Hey @hjuutilainen 

When I'm running SUS Inspector version 0.3.1 (0.3.1) I do get the following error message when trying to refresh catalogs as well as on the initial sync:
```bash
Task exited with code 1.

Traceback (most recent call last):
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 835, in <module>
    main()
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 831, in main
    download_packages=download_packages)
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 733, in sync
    copy_only_if_missing=fast_scan)
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 550, in replicateURLtoFilesystem
    getURL(full_url, local_file_path)
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 465, in getURL
    onlyifnewer=True, etag=saved_etag)
  File "/Users/myuser/Library/Application Support/SUS Inspector/Reposado/reposado.bundle/code/repo_sync", line 349, in curl
    header['http_result_description'] = line.split(None, 2)[2]
IndexError: list index out of range
```

Resetting SUS Inspector as decribed [here](https://github.com/hjuutilainen/sus-inspector#resetting-and-uninstalling) did not resolve the issue.

This appeares to be an error with Reposado. However, I did set up reposado withouth SUS inspector on the same machine and it is running withouth issues. Is it maybe an issue with the version of Reposado that has been bundled with SUS Inspector?

I'm running macOS 10.14.5 and I'm using the default SUS provided by Apple. Here's the relevant output from `com.hjuutilainen.SUS-Inspector.plist`
```
baseURL = "https://swscan.apple.com/content/catalogs/others/index-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog";
```

Thank you very much for your reply.
@aschwanb

**Comment by hjuutilainen** [comment]
Thanks for reporting. Confirmed here too and seems to be related to the bundled reposado version. Fixing soon.

**Comment by hjuutilainen** [comment]
Version 0.3.2 now available with updated Reposado.

**Comment by aschwanb** [comment]
Thank you very much for your fast reply. I can confirm that the issue is resolved in 0.3.2
👍
